coap/examples/coap_client/main/idf_component.yml

```diff
@@ -1,8 +1,9 @@
-version: "1.0.0"
+version: 1.0.0
 description: CoAP Client Example
 dependencies:
   espressif/coap:
-    version: "^4.3.0"
-    override_path: '../../../'
+    version: ^4.3.0
+    override_path: ../../../
   protocol_examples_common:
     path: ${IDF_PATH}/examples/common_components/protocol_examples_common
+  espressif/ethernet_init: '*'
```

Comment:
This should not be required, `protocol_examples_common` should internally pull in this dependency. Are you getting some build error without this change?

Reply:
I did bump version in `coap/idf_component.yml` later on - which could have been after I did `idf.py add-dependency espressif/ethernet_init`.